scripts/receive_meteor.sh

```diff
@@ -176,59 +176,50 @@ if [ "$METEOR_RECEIVER" == "rtl_fm" ]; then
     fi
   fi
 
-  log "Decoding in progress (QPSK to BMP)" "INFO"
-  ${IMAGE_PROC_DIR}/meteor_decode_qpsk.sh "${qpsk_file}" "${AUDIO_FILE_BASE}" >> $NOAA_LOG 2>&1
-  
-  sleep 10
+  log "Running MeteorDemod do demodulate QPSK file, rectify (spread) images, create heat map and composites and convert them to JPG" "INFO"
+  $METEORDEMOD -t "$TLE_FILE" -f jpg -i "${qpsk_file}" >> $NOAA_LOG 2>&1 #Dodao sam >> $NOAA_LOG 2>&1
 
-  rm "${qpsk_file}"
+  log "Removing QPSK, GCP and BMP files" "INFO"
+  rm "${qpsk_file}" *.gcp *.bmp
 
-  if [ -f "${AUDIO_FILE_BASE}.dec" ]; then
-    if [ "${SUN_ELEV}" -lt "${SUN_MIN_ELEV}" ]; then
-      log "I got a successful ${FILENAME_BASE}.dec file. Decoding APID 68" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_apid68_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-      $CONVERT $FLIP -negate "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    else
-      log "I got a successful ${FILENAME_BASE}.dec file. Creating false color image" "INFO"
-      ${IMAGE_PROC_DIR}/meteor_false_color_decode.sh "${AUDIO_FILE_BASE}.dec" "${IMAGE_FILE_BASE}-122" >> $NOAA_LOG 2>&1
-	  $CONVERT $FLIP "${IMAGE_FILE_BASE}-122.bmp" "${IMAGE_FILE_BASE}-122.bmp" >> $NOAA_LOG 2>&1
-    fi
+  log "Annotating images and creating thumbnails" "INFO"
+  counter=1
+  for i in *.jpg
+    do
+      ${IMAGE_PROC_DIR}/meteor_normalize_annotate.sh "$i" "$i" 100 >> $NOAA_LOG 2>&1
```

Comment:
Is the normalisation step this script performs still required when using meteordemod?

Reply:
Well, it annotates the received Meteor image.

Reply:
Oh I thought it also did another thing, my bad.